frontend/src/components/annotation/annotation_core.vue

```diff
@@ -1089,7 +1089,8 @@ export default Vue.extend( {
       },
 
       annotations_loading: false,
-      save_loading: false,
+      save_loading_image: false,
```

Comment:
Is the plan in the future to have concrete flags for text and audio files? If the cases are different enough maybe that's more readable to have it concrete but just double checking

Reply:
I thin for future file types, I would prefer to move in the direction of having a completely different component built from scratch, since there are still many gotchas and things we don't know yet and too keep bugs and feature changes as isolated as possible. Maybe things that are bery generic, can me moved into a class to be reused by both but in general I would prefer to have separate saves flags and even a completely different componenet for audio/text/lidar for example.